pkg/apk/implementation.go

```diff
@@ -393,6 +393,15 @@ func (a *APK) InitKeyring(ctx context.Context, keyFiles, extraKeyFiles []string)
 				if err != nil {
 					return fmt.Errorf("failed to read apk key: %w", err)
 				}
+			case "s3": //nolint:goconst
```

Comment:
This seems interesting, but I'm a bit worried about adding features that might move compatibility away from `apk add` and other non-go-apk tools.